parm/atm/obs/config/amsua_n19.yaml

```diff
@@ -0,0 +1,323 @@
+obs space:
+  name: amsua_n19
+  obsdatain:
+    obsfile: $(OBS_DIR)/$(OBS_PREFIX).amsua_n19.$(OBS_DATE).nc4
+  obsdataout:
+    obsfile: $(DIAG_DIR)/diag_amsua_n19_$(OBS_DATE).nc4
+  simulated variables: [brightness_temperature]
+  channels: &amsua_n19_channels 1-15
+obs operator:
+  name: CRTM
+  Absorbers: [H2O,O3,CO2]
+  Clouds: [Water, Ice]
+  Cloud_Fraction: 1.0
+  obs options:
+    Sensor_ID: amsua_n19
+    EndianType: little_endian
+    CoefficientPath: $(CRTM_COEFF_DIR)
+obs bias:
+  input file: $(BIAS_DIR)/$(BIAS_PREFIX).amsua_n19.satbias.$(BIAS_DATE).nc4
+  variational bc:
+    predictors:
+    - name: constant
+    - name: lapse_rate
+      order: 2
+      tlapse: &amsua_n19_tlapse $(BIAS_DIR)/$(BIAS_PREFIX).amsua_n19.tlapse.$(BIAS_DATE).txt
```

Comment:
Based on comments in ufo issues tlapse is usually treated as a constant.   We can leave it as a cycled file for the time being but the file contents probably won't be evolving with time.

Reply:
This point need not hold up this PR.   Mark as resolved and address in subsequent PRs.